client/src/utils/api.ts

```diff
@@ -44,6 +44,18 @@ export class users extends pbAPI {
   get(id: UserType['id'], query?: BaseQueryParams) {
     return this.pb.collection('users').getOne<UserType>(id, query)
   }
+
+  edit(
+    id: UserType['id'], 
+    username: UserType['username'],
+    email: UserType['email']
+  ) {
+    const data = {
+      username,
+      email
```

Comment:
~~Should we really allow users to change their email?~~ Ok so apparently users [can't change their system field](https://github.com/pocketbase/pocketbase/discussions/1032#discussion-4568778) unless they have an admin permission. Anyways, for now they should just be able to change their name and profile pic - that's going to be a separate ticket though.

Also, we are using the `name` field, not the `username` field (`username` is populated by default while `name` isn't). `name` has been used all across the website already so that's just a simple fix for this method.
![image](https://github.com/WSU-Society-of-Computer-Developers/summer-project/assets/35277113/524d29f0-dc81-43b3-9adc-f721c4732d6a)

Reply:
All done! I changed the method in the api, and updated the user page to match.